qdrant-landing/content/documentation/frameworks/langchain4j.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Langchain4J
+weight: 2100
+---
+
+# Langchain4J
+
+[Langchain4J](https://github.com/langchain4j/langchain4j) is a community port of [Langchain](https://www.langchain.com/) for building context-aware AI applications in Java
+
+You can use Qdrant as a vectorstore in Langchain4J through the [`langchain4j-qdrant`](https://central.sonatype.com/artifact/dev.langchain4j/langchain4j-qdrant) module.
+
+## Setup
+
+Add the `langchain4j-qdrant` to your project dependencies.
+
+```xml
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-qdrant</artifactId>
+    <version>VERSION</version>
+</dependency>
+```
+
+## Usage
+
+```java
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.qdrant.QdrantEmbeddingStore;
+
+EmbeddingStore<TextSegment> embeddingStore =
+    QdrantEmbeddingStore.builder()
+        // Ensure the collection is configured with the appropriate dimensions
+        // of the embedding model.
+        // Reference https://qdrant.tech/documentation/concepts/collections/
+        .collectionName("{collection_name}")
+        .host("xyz-example.eu-central.aws.cloud.qdrant.io")
+        // GRPC port of the Qdrant server
+        .port(6334)
+        .apiKey("<YOUR_API_KEY")
```

Comment:
I think we should have some consistency with the variables that we want readers to substitute for. And is `.host` limited to cloud hosts, or would it work with `http://localhost:6333`, based on the Quick Start guide (aka "YOUR_QDRANT_REST_URL," from the other file?)

So maybe:

(If you want to set it up in a different way, I'm good with that too. Long-term, I'd like to set up Hugo to substitute for those variables.)

```suggestion
        .collectionName("YOUR_COLLECTION_NAME")
        .host("YOUR_HOST_URL")
        // GRPC port of the Qdrant server
        .port(6334)
        .apiKey("YOUR_API_KEY")
```